go-controller/pkg/metrics/ovnkube_controller.go

```diff
@@ -284,6 +285,26 @@ var metricBANPCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "The total number of baseline admin network policies (0 or 1) in the cluster",
 })
 
+var metricANPDBObjects = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	Namespace: MetricOvnkubeNamespace,
+	Subsystem: MetricOvnkubeSubsystemController,
+	Name:      "admin_network_policy_custom_resource_nbdb_objects_total",
```

Comment:
Not sure if `custom_resource` part adds a lot of value, wdyt?
also, thinking how we would like to expose this metrics to make it easier to understand, maybe `nbdb_objects` -> `db_objects`?

Reply:
I think martin asked to add custom_resource in grand scheme of events but let's see what he says and I can then accordingly also update the other two metrics we had for count.

Reply:
dont have a strong opinion on whether to add `...custom_resource_` or not.

Reply:
ack ok then I am changing this to `admin_network_policy_db_objects_total`